modules/ROOT/pages/deployment/services/s-list/auth-service.adoc

```diff
@@ -0,0 +1,30 @@
+= Auth Service Configuration
+:toc: right
+
+:service_name: auth-service
+:no_second_tab: true
+//:no_third_tab: true
+
+:description: The Infinite Scale Auth Service service is used to authenticate service accounts. Compared to normal accounts, service accounts are Infinite Scale internal only and not available as ordinary users like via LDAP.
+
+== Introduction
+
+{description}
+
+== Default Values
+
+* Auth Server listens on port 9198 by default.
+
+include::partial$deployment/services/auth-service-family.adoc[]
+
+== Service Accounts
+
+Service accounts are user accounts that are only used for inter service communication. The users have no personal space, do not show up in user lists and cannot login via the UI. Service accounts can be configured in the settings service. Only the `admin` service user is available for now. Additionally to the actions it can do via its role, all service users can stat all files on all spaces.
```

Comment:
```suggestion
Service accounts are user accounts that are only used for inter-service communication. These users have no personal space, do not show up in user lists and cannot login via the UI. Service accounts can be configured in the settings service. Only the `admin` service user is available for now. In addition to the actions it can do via its role, all service users can stat all files on all spaces.
```